### DISCUSSION TITLE
can Luban be used to control GRBL machines?

### DISCUSSION THREAD

**Discussion by rebeltaz** [Snapmaker/Luban#2156] [Q&A]
A maker on instructables uses Luban for the designs he creates. The process is a little different than when using Lightburn. I was just wondering if Luban could be used to control GRBL machines? I'd be interested in giving it a spin if it was possible.

**Comment by parachvte** [comment]
Currently we support G-code generation in Marlin flavor G-code.

GRBL is too old. I'm not sure if all G-code works on a GRBL machine.

**Reply by rmeissn** [reply]
I'd also be interested to export carving files in a grbl format. Luban must not control the machine itself, but producing grbl compatible files would be nice!
